dice_ml/explainer_interfaces/dice_tensorflow1.py

```diff
@@ -85,6 +85,7 @@ def generate_counterfactuals(self, query_instance, total_CFs, desired_class="opp
         :param tie_random: Used in rounding off CFs and intermediate projection.
         :param stopping_threshold: Minimum threshold for counterfactuals target class probability.
         :param posthoc_sparsity_param: Parameter for the post-hoc operation on continuous features to enhance sparsity.
+        :param posthoc_sparsity_algorithm: Perform either linear or binary search. Takes "linear" or "binary".
```

Comment:
say more in the docs to introduce the difference b/w linear and binary, and when binary search can be used.

Reply:
added